simple_cb/simple_cb_magma.py

```diff
@@ -47,18 +47,17 @@ class _SimpleCB(m.Circuit):
             "config_en", m.In(m.Bit),
             "read_data", m.Out(m.Bits(CONFIG_DATA_WIDTH))
         ]
-        IO += m.ClockInterface(has_reset=True)
+        IO += m.ClockInterface(has_async_reset=True)
 
         @classmethod
         def definition(io):
             config = mantle.Register(CONFIG_DATA_WIDTH,
                                      init=config_reset,
                                      has_ce=True,
-                                     has_reset=True)
+                                     has_async_reset=True)
 
             config_addr_zero = mantle.eq(m.uint(0, 8), io.config_addr[24:32])
-            config(io.config_data, reset=io.RESET,
```

Comment:
why do we remove the reset signals going into the reg?

Reply:
Reset (and clock types more generally) have a default wiring semantics in the compiler. If there's an unwired reset port and a reset on the interface, it will wire them up.